src/interfaces.d.ts

```diff
@@ -1,21 +1,30 @@
-export interface Column<T> {
+import { DNode } from '@dojo/widget-core/interfaces';
+
+export interface Column<T = any, V = string> {
```

Comment:
We should have some documentation for exported interfaces, will help explain the intent of the properties and functions.